gitlab/Caddyfile-socket

```diff
@@ -0,0 +1,24 @@
+https://dev.sth.se.k0nsl.org {
```

Comment:
Ah, yes! Mea culpa. I knew we would have to sort out these things later on; I simply submitted the file "as is".